velox/functions/prestosql/aggregates/ApproxPercentileAggregate.cpp

```diff
@@ -650,7 +663,8 @@ class ApproxPercentileAggregate : public exec::Aggregate {
     if constexpr (checkIntermediateInputs) {
       VELOX_USER_CHECK(rowVec);
       for (int i = kPercentiles; i <= kAccuracy; ++i) {
-        VELOX_USER_CHECK(rowVec->childAt(i)->isConstantEncoding());
+        VELOX_USER_CHECK(isConstantVector(
```

Comment:
Unrelated change? Would you extract it into a separate PR, provide clear description and tests?

Reply:
It was related since without the change AggregateFuzzer would fail. 

However I'll try reverting the change since the failure might be related to flushing and spilling was once both enabled in the PR. Probably it's no longer needed.